lms/djangoapps/courseware/courses.py

```diff
@@ -413,7 +415,9 @@ def get_course_date_blocks(course, user, request=None, include_past_dates=False,
     if DATE_WIDGET_V2_FLAG.is_enabled(course.id):
         blocks.append(CourseExpiredDate(course, user))
         blocks.extend(get_course_assignment_due_dates(
-            course, user, request, num_return=num_assignments, include_past_dates=include_past_dates))
+            course, user, request, num_return=num_assignments,
+            include_past_dates=include_past_dates, include_access=True
```

Comment:
Do we want `include_access` to be True on this method? Since this function is called on the date widget in the course outline, we would be doing that additional check without using any of that data in that view.

Reply:
Bad rebase, just gonna pass through from the kwargs

Reply:
Awesome, thanks